Sources/DailyChallengeFeature/DailyChallengeView.swift

```diff
@@ -11,7 +11,7 @@ import SwiftUI
 
 public struct DailyChallengeReducer: ReducerProtocol {
   public struct State: Equatable {
-    public var alert: AlertState<Action>?
+    @PresentationState public var alert: AlertState<Action.Alert>?
```

Comment:
modernized some alerts to get rid of the deprecated `alert(_:dismiss:)` view modifier.